chargin_station_test.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+import os
+from numpy.random import randint
+from charging_station import ChargingStation
+
+
+CS = ChargingStation()

```

Comment:
it is more standard to use lower case, explicit names for an instance of a class
```suggestion
charging_station = ChargingStation()
```